src/Sprites/ImageType.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <cstdint>
+
+enum class ImageType : uint16_t
+{
+	MenuGraphic = 0,
+	GameGraphic = 1,
+	UnitShadow1 = 4,
+	UnitShadow2 = 5,
+	Unknown1 = 33,
+	TruckBed = 65,
+	Unknown3 = 97,
+	Unknown4 = 129
+};
```

Comment:
I think you mean enum instead of struct in your comment? 

Nice deductive work. I pulled those values out of ArtFile by tabulating all recorded values for the uint16_t in the existing .prt file. Maybe now that we know it is a bitfield, we could figure out what some of the other values mean. 

I'll update the code base to be this bitfield shortly.